gateway-ha/src/main/java/com/lyft/data/gateway/ha/persistence/dao/Selector.java

```diff
@@ -0,0 +1,86 @@
+package com.lyft.data.gateway.ha.persistence.dao;
+
+import static com.lyft.data.gateway.ha.router.PrestoResourceManager.SelectorDetail;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.javalite.activejdbc.Model;
+import org.javalite.activejdbc.annotations.BelongsTo;
+import org.javalite.activejdbc.annotations.Cached;
+import org.javalite.activejdbc.annotations.IdName;
+import org.javalite.activejdbc.annotations.Table;
+
+@BelongsTo(parent = ResourceGroup.class, foreignKeyName = "resource_group_id")
+@IdName("resource_group_id")
+@Table("selectors")
+@Cached
+public class Selector extends Model {
```

Comment:
The class name should match with the table name. Therefore, it should be named `Selectors`